src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md

```diff
@@ -150,7 +151,27 @@ const KittenBase = kind({ /* ... */ });
 const Kitten = Spottable(KittenBase);
 
 export default Kitten;
-export {Kitten, KittenBase};
+export {
+	Kitten, 
+	KittenBase
+};
+```
+As well as wrapping your component, we need to define the color of Spotlight effect. We'll use light-grey as the background color of the Spotlight, and black as the color of the text covered with Spotlight.
+
+**./src/components/Kitten/Kitten.module.less**
+```css
+@import "~@enact/sandstone/styles/mixins.less";
+
+.kitten {
+	display: inline-block;
+	padding: 22px;
+	text-align: center;
+	background-color: transparent;
+	.focus({
+		background-color:  #e6e6e6; // light-grey
+		color: black;
+	});
+}
```

Comment:
I think this block should be end of this subject, after the "advanced" block.

Reply:
And we need more information about using `focus` mixin.

Reply:
OK. 
I moved it after the "advanced" block and added more info about 'focus' mixin